src/commandlineflags.h

```diff
@@ -11,28 +11,33 @@
 #define FLAG(name) FLAGS_##name
 
 // Macros for declaring flags.
+// NOLINTBEGIN
```

Comment:
IMHO it's best to specify the specific check in parenthesis, but maybe that doesn't work for in scoped case?